<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sys_New_Project_Dia</class>
 <widget class="QDialog" name="Sys_New_Project_Dia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>889</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>660</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>2000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Project</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/system_sys_resource.qrc">
    <normaloff>:/prom_icon</normaloff>:/prom_icon</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="toolTip">
        <string>The name of the project file/directory) is equal to the project name, with the suffix *.prm. If the name exits already in the directory or in the database the name will be *_1.prm. No capital letter are permitted!</string>
       </property>
       <property name="text">
        <string>Projectname</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_pro_name">
       <property name="toolTip">
        <string>The projectname is limited to 29 signs!</string>
       </property>
       <property name="maxLength">
        <number>29</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Author         </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_author">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_database">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>All project requires a database connection, where the data (input, results) are stroed  </string>
     </property>
     <property name="title">
      <string>Database connection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_db_is_set">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Connection is set</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_set_db">
        <property name="text">
         <string>&amp;Set Connection...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_db_is_checked">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Connection is checked / ok</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_check_db">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Check Connection</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_pro_type">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Projecttype</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="5">
       <widget class="QRadioButton" name="radioButton_hydrol">
        <property name="text">
         <string>HYDROL</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QRadioButton" name="radioButton_hyd">
        <property name="text">
         <string>HYD</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="radioButton_hyd_file">
        <property name="text">
         <string>HYD_File</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QRadioButton" name="radioButton_hyd_dam">
        <property name="text">
         <string>HYD+DAM</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_all_types">
        <property name="text">
         <string>ALL</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioButton_risk_typ">
        <property name="text">
         <string>RISK</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="radioButton_dam_typ">
        <property name="text">
         <string>DAM</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QRadioButton" name="radioButton_fpl_typ">
        <property name="text">
         <string>FPL</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QRadioButton" name="radioButton_hyd_temp">
        <property name="text">
         <string>HYD-TEMP (exp)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_path">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Path</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="Sys_One_Filechooser_Wid" name="widget_browser" native="true">
        <property name="minimumSize">
         <size>
          <width>329</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_logfile">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>60</height>
      </size>
     </property>
     <property name="title">
      <string>Logfile</string>
     </property>
     <widget class="QCheckBox" name="checkBox_logfile">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>201</width>
        <height>19</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Save logfiles in logfile archive</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_descript">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Description</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextEdit" name="textEdit_descript"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>209</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Sys_One_Filechooser_Wid</class>
   <extends>QWidget</extends>
   <header location="global">sys_one_filechooser_wid.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/system_sys_resource.qrc"/>
 </resources>
 <connections/>
</ui>
